docs/advanced-guides/posts.md

```diff
@@ -364,16 +364,28 @@ The blob object, as JSON, would look something like:
 }
 ```
 
-The blob is then included in a `app.bsky.embed.images` array, along with an alt-text string. The `alt` field is required for each image. Pass an empty string if there is no alt text available.
+The blob is then included in a `app.bsky.embed.images` array, along with an alt-text string and the image dimensions.
+
+The `alt` field is required for each image. Pass an empty string if there is no alt text available.
+
+To get the image dimensions, you'll need a library like [Pillow](https://pillow.readthedocs.io/). Client apps need the image dimensions to display them without performance or visual issues.
```

Comment:
I would add some text like "The specific dimensions don't need to be known, it is the height/width ratio which is important. If you don't know the image aspect ratio and don't parse the image to detect the dimensions, it is best to leave the `aspectRatio` field undefined instead of guessing."

Reply:
good call